doc/man3/OSSL_ENCODER.pod

```diff
@@ -74,6 +76,8 @@ algorithm that's identifiable with I<name>.
 OSSL_ENCODER_number() returns the internal dynamic number assigned to
 the given I<encoder>.
 
+OSSL_ENCODER_number() returns the name used to fetch the given I<encoder>.
```

Comment:
Copy & paste error. Also it is not the name used to fetch but the first name from the provider name list. You could just copy the description from for example EVP_SIGNATURE_name().

Reply:
Yeah, should have looked elsewhere for sample text.  Didn't think of it.